orochi/client.py

```diff
@@ -685,6 +705,12 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            f = open(expanduser("~") + '/.orochi.current_song.log', 'w')
+            f.write(u' '.join((track_name, '|', track_performer)).encode('utf-8').strip())
```

Comment:
A custom format will probably not suit everyone. I first thought of using CSV, but that's not too easy to parse properly in CSV. Instead, I'd propose to use a line based format: Track, Performer, Album, Year, each separated by a newline.

```python
songfile.write('\n'.join([track_name, track_performer, track_album, track_year or ''])
```

This way you can use regular bash tools to retrieve a specific line. For example `head` / `tail`:

```bash
ARTIST=$(cat current_song.txt | head -n 2 | tail -n 1)
```

Or even nicer with `sed`:

```bash
ARTIST=$(sed '2q;d' current_song.txt)
```